bookkeeper-http/http-server/src/main/java/org/apache/bookkeeper/http/HttpRouter.java

```diff
@@ -33,14 +33,14 @@
   public static final String HEARTBEAT                    = "/heartbeat";
   public static final String SERVER_CONFIG                = "/api/v1/config/server_config";
   // bookkeeper
-  public static final String LIST_BOOKIES                 = "/api/v1/bookkeeper/list_bookies";
-  public static final String UPDATE_COOKIE                = "/api/v1/bookkeeper/update_cookie";
+
   // ledger
   public static final String DELETE_LEDGER                = "/api/v1/ledger/delete";
   public static final String LIST_LEDGER                  = "/api/v1/ledger/list";
   public static final String GET_LEDGER_META              = "/api/v1/ledger/metadata";
   public static final String READ_LEDGER_ENTRY            = "/api/v1/ledger/read";
   // bookie
+  public static final String LIST_BOOKIES                 = "/api/v1/bookie/list_bookies";
```

Comment:
This isn't a resource of the bookie, but rather a resource of the cluster

Reply:
Thanks. right. it is first placed under bookkeeper, but there seems only this endpoint left, so moved here, since this is bookie related. :)